### PR TITLE
Update WarpDrive notebook with fixes

### PR DESCRIPTION
**Pull request by kaushikb11**
## What does this PR do?

Follow up to #144 

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: lightning_examples/warp-drive/multi_agent_rl.py [hunk 1]
```diff
@@ -8,32 +8,44 @@
...
+# %%
+import os
+import sys
+
+_IN_COLAB = "google.colab" in sys.modules
+
+if _IN_COLAB:
+    os.system("git clone https://github.com/salesforce/warp-drive.git")
+    os.chdir("warp-drive")
+    os.system("pip install -e .")
```
Comment (Borda) [inline lightning_examples/warp-drive/multi_agent_rl.py:31]: why install master? then it is not reproducible...

**Review by Borda — CHANGES_REQUESTED** [review]

File: lightning_examples/warp-drive/multi_agent_rl.py (hunk 1 above)
Comment (kaushikb11) [reply lightning_examples/warp-drive/multi_agent_rl.py:31]: Need to access https://github.com/salesforce/warp-drive/tree/master/example_envs

Comment (sunil-s) [reply lightning_examples/warp-drive/multi_agent_rl.py:31]: Thanks for pointing this out, @Borda . Seems like I did add example envs as part of the package. So, let me just have the regular pip-install then

Comment (kaushikb11) [reply lightning_examples/warp-drive/multi_agent_rl.py:31]: @sunil-s How would we access `example_envs` then? Mind sharing sample code?

Comment (sunil-s) [reply lightning_examples/warp-drive/multi_agent_rl.py:31]: Just the way before `from example_envs.tag_continuous.tag_continuous import TagContinuous` should work

**Pull request merged**